react_week_1.md

```diff
@@ -0,0 +1,52 @@
+# Week 1
+
+## React. Components
+
+Should be done:
+1.	Create separate branch for this task.
+2.	Use CRA with template --typescript.
+3.	Set up eslint and prettier so that project would not build if having errors. Use [configs](https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/module01/configs.md) [RU]
+4.	Add React-Router version 6. Add header showing current page, also add pages “About Us”, “404”. If trying to input unknown route into url – application should redirect to “404”.
+5.	Draw the following on the main page:
+* Search Bar.\
+examples:\
+https://studio.uxpincdn.com/studio/wp-content/uploads/2020/09/BlogHeader_SearchBar_1200x600.png \
+https://www.sliderrevolution.com/wp-content/uploads/2021/02/cssheader1.jpg \
+Input value should be saved to LocalStorage during component’s unmount. While initializing look at it's value in LocalStorage and show it.
```

Comment:
The input value should be saved to LocalStorage during the component’s unmount. While initializing look at it is value in LocalStorage and show it.